README.md

```diff
@@ -1310,6 +1311,42 @@ Jwts.parser()
     // ... etc ...
 ```
 
+<a name="json-jackson-custom-parsing"></a>
+### Parsing of Custom Claim Types
+
+By default JJWT will only convert simple claim types: String, Date, Long, Integer, Short and Byte.  If you need to deserialize other types you can configure the `JacksonDeserializer` by passing a `Map` of claim names to types in through a constructor. For example:
+
+```java
+new JacksonDeserializer(Maps.of("user", User.class))
+```
+
+This would trigger the value in the `user` claim to be deserialized into the custom type of `User`.  Given the claims body of:
+
+```json
+{
+    "issuer": "https://example.com/issuer",
+    "user": {
+      "firstName": "Jill",
+      "lastName": "Coder"
+    }
+}
+```
+
+The `User` object could be retrieved from the `user` claim with the following code:
+
+```java
+Jwts.parserBuilder()
+
+    .deserializeJsonWith(new JacksonDeserializer(Maps.of("user", User.class)))
+
+    .build()
+    .parseClaimsJwt(aJwtString)
+    .build()
+    .getBody()
+    
```

Comment:
Blank?

Reply:
I was trying to follow what I thought was the existing convention, i.e.:
```java
Jwts.parser()

    .setSigningKeyResolver(signingKeyResolver) // <----
    
    .parseClaimsJws(jwsString);
```

My personal pref is would be to strip those empty lines

Reply:
Ah, I see what we did - it was to make it more visually obvious as to what is calling attention.  Let's stick to the current convention, and we can have another PR if we want to change that convention.

Now that I'm looking at it, the extra lines with the `// <----- ` makes it more visually obvious and probably enhances readability.  Just my .02 though.